src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -27,6 +27,7 @@ import {
 } from "utils/selectors";
 import { eventEndTime } from "utils/event";
 import { showZendeskWidget } from "utils/zendesk";
+import { RenderMarkdown } from "components/organisms/RenderMarkdown";
```

Comment:
This should be sorted down with all of the other `components/organisms/*` imports

Reply:
done